contracts/prize-strategy/PrizeStrategyStorage.sol

```diff
@@ -34,7 +39,7 @@ contract PrizeStrategyStorage {
   uint256 public prizeAverageTickets;
 
   RNGInterface public rng;
-  uint256 public rngRequestId;
+  RngRequest internal rngRequest;
```

Comment:
It would be helpful to have an external accessor so that we can know which block it will lock

Reply:
added:
```
function getLastRngLockBlock() public view returns (uint32) { ... }
function getLastRngRequestId() public view returns (uint32) { ... }
```